src/sessions.ts

```diff
@@ -761,22 +771,22 @@ function applySession(
   session: ClientSession,
   command: Document,
   options?: CommandOptions
-): MongoError | undefined {
+): MongoDriverError | undefined {
   // TODO: merge this with `assertAlive`, did not want to throw a try/catch here
   if (session.hasEnded) {
-    return new MongoError('Attemped to use a session that has ended');
+    return new MongoDriverError('Attemped to use a session that has ended');
```

Comment:
clean up request: `Attemped` -> `Attempted`

Reply:
Nice catch, I wasn't even looking at the messages themselves